Formula/yuck.rb

```diff
@@ -0,0 +1,34 @@
+class Yuck < Formula
+  desc "Bog-standard command-line option parser for C"
+  homepage "http://www.fresse.org/yuck/"
+  url "https://github.com/hroptatyr/yuck/releases/download/v0.2.2/yuck-0.2.2.tar.xz"
+  sha256 "9cd0c1af9a4afe6002f1672c8ece91d305cea1f61fbc07c7ce94a0bbe634deec"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.yuck").write("Usage: foo")
+    system "#{bin}/yuck", "gen", "test.yuck"
+  end
+
+  patch :p0, :DATA
```

Comment:
Please submit this patch upstream, add a comment referencing the upstream PR, and ideally replace this with a `patch do` block (should come before the first `depends_on`) that references the upstream commit. We don't accept patches that haven't been accepted/confirmed by upstream.